src/ocean/text/convert/DateTime_tango.d

```diff
@@ -586,11 +604,9 @@ struct DateTimeLocale
         return f;
     }
 
-    /**********************************************************************
-
-     **********************************************************************/
-
-    private char[] formatCustom (ref Result result, Time dateTime, cstring format)
+    /// Implementation of format
```

Comment:
I'd rather keep any cleanup for another day (I prefer to do all at once). Will revert :)